demo/dotnet/README.md

```diff
@@ -97,26 +92,28 @@ of debugging facilities baked into the demo application to solve this. First, ty
 dotnet run -c MicDemo.Release -- --show_audio_devices
 ```
 
-It provides information about various audio input devices on the box. On a Windows PC, this is the output:
+It provides information about various audio input devices on the box. This is an example of the output:
 
 ```
-Available input devices:
-
-    Device 0: Microphone Array (Realtek(R) Au
-    Device 1: Microphone Headset USB	
+index: 0, device name: Monitor of sof-hda-dsp HDMI3/DP3 Output
+index: 1, device name: Monitor of sof-hda-dsp HDMI2/DP2 Output
+index: 2, device name: Monitor of sof-hda-dsp HDMI1/DP1 Output
+index: 3, device name: Monitor of sof-hda-dsp Speaker + Headphones
+index: 4, device name: sof-hda-dsp Headset Mono Microphone + Headphones Stereo Microphone
+index: 5, device name: sof-hda-dsp Digital Microphone	
```

Comment:
Is this what the output usually looks like? There are more devices here than I would expect - I only usually get two on my laptop